MessageBusProcessor.php

```diff
@@ -1,64 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Enqueue\MessengerAdapter;
-
-use Interop\Queue\PsrContext;
-use Interop\Queue\PsrMessage;
-use Interop\Queue\PsrProcessor;
-use Symfony\Component\Messenger\Asynchronous\Transport\ReceivedMessage;
-use Symfony\Component\Messenger\MessageBusInterface;
-use Symfony\Component\Messenger\Transport\Serialization\DecoderInterface;
-use Enqueue\MessengerAdapter\Exception\RejectMessageException;
-use Enqueue\MessengerAdapter\Exception\RequeueMessageException;
-
-/**
- * The processor could be used with any queue interop compatible consumer, for example Enqueue's QueueConsumer.
- *
- * @author Max Kotliar <kotlyar.maksim@gmail.com>
- * @author Samuel Roze <samuel.roze@gmail.com>
- */
-class MessageBusProcessor implements PsrProcessor
```

Comment:
This should not be removed. This allows using Enqueue's consume command as a receiver. Why did you think it should be removed?

Reply:
Because `MessageBusProcessor` does not integrate with Enqueue's consume and `Symfony messenger`:

1. So that `MessageBusProcessor` is called, you must pass the following properties to the message: `enqueue.topic_name`and  `enqueue.processor_name`. Where is it better to set these properties, in `QueueInteropTransport`?

1. `ReceivedMessage` does not take arguments in `__construct`, but `MessageBusProcessor` transmits them.

I can revert `MessageBusProcessor` as is, but it will not work, and try fixed in new pull request.